base/setup/usetup/lang/es-ES.h

```diff
@@ -136,61 +146,61 @@ static MUI_ENTRY esESIntroPageEntries[] =
     {
         4,
         3,
-        " ReactOS " KERNEL_VERSION_STR " Setup ",
+        " Instalaci�n de ReactOS " KERNEL_VERSION_STR " ",
         TEXT_STYLE_UNDERLINE
     },
     {
         6,
         8,
-        "ReactOS Version Status",
+        "Acerca del estado de ReactOS",
         TEXT_STYLE_HIGHLIGHT
     },
     {
         6,
         11,
-        "ReactOS is in Alpha stage, meaning it is not feature-complete",
+        "ReactOS est� en estado Alpha, es decir, no es un sistema completo",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         12,
-        "and is under heavy development. It is recommended to use it only for",
+        "y est� en constante desarrollo. S�lo es recomendado usarlo con fines",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         13,
-        "evaluation and testing purposes and not as your daily-usage OS.",
+        "de prueba y evaluaci�n y no como su sistema de uso diario.",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         15,
-        "Backup your data or test on a secondary computer if you attempt",
+        "Respalde sus datos o pru�belo en una computadora no habitual si usted",
```

Comment:
Should there be a space after the comma?

Reply:
Char 130 corresponds to `&sbquo;` in iso-8859-1 but in CP437 corresponds to "é" (e with accent)